api/src/main/java/de/oliver/fancynpcs/api/events/NpcCreateEvent.java

```diff
@@ -16,12 +17,12 @@ public class NpcCreateEvent extends Event implements Cancellable {
     @NotNull
     private final Npc npc;
     @NotNull
-    private final Player player;
+    private final CommandSender creator;
```

Comment:
I think only players should be able to create npcs

Reply:
This is only the event, not the command, I do believe in the CreateCMD class that I limited it only to players